prusti-viper/src/encoder/middle/core_proof/snapshots/into_snapshot/common/mod.rs

```diff
@@ -455,13 +456,30 @@ pub(super) trait IntoSnapshotLowerer<'p, 'v: 'p, 'tcx: 'v> {
             .collect::<Result<Vec<_>, _>>()?;
         let mut args =
             self.expression_vec_to_snapshot(lowerer, &app.arguments, expect_math_bool)?;
+        log::debug!(
+            "BFASNAPSHOT  ty_args:{ty_args:?} app_args:{:?} args:{args:?}",
```

Comment:
What does `BFASNAPSHOT` mean?

Reply:
It was a temporary debug statement to find out where type and expression inconsistencies are introduced.
I removed it.